api/Navigator.json

```diff
@@ -2974,21 +2974,13 @@
           "support": {
             "chrome": {
               "version_added": "13",
-              "notes": [
-                "Allowed schemes include <code>mailto</code>, <code>mms</code>, <code>nntp</code>, <code>rtsp</code>, and <code>webcal</code>. Custom protocols must be prefixed with <code>web+</code>.",
```

Comment:
Note, 
- schemes now captured as subfeatures. Though might need to add rtsp back if you say (this note indicated supported from v13, but tests show not supported in v15)
- Custom protocol note not needed - that is a spec requirement so this doesn't indicate an incompatibility.